distribution/ddf-common/src/main/resources/etc/org.apache.karaf.shell.cfg

```diff
@@ -0,0 +1,13 @@
+#
+# Configuration for SSH shell.
+# This configuration is turned off by default; in order to turn it on, uncomment
+# the sshPort property.
+# When the port is available and active, SSH access will be restricted to connections
+# originating from the same host machine. To allow direct remote access to the shell,
+# change the value of the sshRealm property to 0.0.0.0
+#
+
+#sshPort=8101
```

Comment:
Does this affect SSHing into the itests? Should SSH be enabled by default for the itests? Should this section https://github.com/codice/ddf/blob/dc42d1bcf665fb4277fa5a3fc7d842ddc9749126/distribution/test/itests/README.md#L19-L24 be updated?

Reply:
It does not effect shelling in for itests. We already handle that for dynamic port assignment here: https://github.com/codice/ddf/blob/master/distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java#L479-L481